inc/breadcrumb.php

```diff
@@ -352,6 +352,12 @@ protected function add_items() {
                 }
             }
 
+		$custom_breadcrumbs = apply_filters( 'dci_get_breadcrumb_items', $this->items );
```

Comment:
```suggestion
		$custom_breadcrumbs = apply_filters( 'dci_get_breadcrumb_items', false, $this->items );
``` 
È sfuggito un problema: se l'oggetto del filtro è `$this->items`, il "corto circuito" si chiude sempre, e l'elaborazione non procede mai! Bisogna passare un booleano invece.
([Problema rilevato qui](https://forum.italia.it/t/breadcrump-non-funzionanti-v1-8-0/38786))